core/templates/dev/head/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -0,0 +1,86 @@
+<script type="text/ng-template" id="inline/collection_editor_navbar_directive">
+  <div
+    class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak"
+    style="margin-left: 20px;">
+    <!-- Editor -->
+    <li ng-class="{'active': getTabStatuses().active === 'main'}">
+      <a
+        href="#"
+        tooltip="Editor"
+        ng-click="selectMainTab()"
+        class="protractor-test-main-tab">
+        <span class="glyphicon glyphicon-pencil"></span>
+      </a>
+    </li>
+
+    <!-- Preview -->
+    {% if SHOW_COLLECTION_NAVIGATION_TAB_PREVIEW and username %}
```

Comment:
Still strongly suggest dropping it. I predict it will be a long time, if ever, before we bring that in, especially since collections are starting out mostly linear.